### PR TITLE
feat: governance-committee-candidates-alex

### PR DESCRIPTION
**Pull request by AlexsJones**
Updated my information.

### CODE REVIEW

**Review by AloisReitbauer — APPROVED** [review]

File: elections/2022/governance-committee-candidates.md [hunk 1]
```diff
@@ -24,13 +24,16 @@ Description of candidate
 
 <img src="static/alex-jones-picture.png" alt="Alex Jones" height="400" width="400">
```
Comment (thschue) [inline elections/2022/governance-committee-candidates.md:25]:
```suggestion
<img src="https://cd.foundation/wp-content/uploads/sites/78/2020/09/alex-jones.jpeg" alt="Alex Jones" height="400" width="400">
```

**Review by thschue — CHANGES_REQUESTED** [review]
Added a suggestion for a photo (CDF community builders), please accept or add another one ...

**Comment by AlexsJones** [comment]
Hi @thschue this has been updated

**Review by thschue — APPROVED** [review]
lgtm

**Pull request merged**